runtime/doc/lsp.txt

```diff
@@ -1853,10 +1858,11 @@ preview_location({location}, {opts})         *vim.lsp.util.preview_location()*
       definition)
 
     Parameters: ~
-      • {location}  a single `Location` or `LocationLink`
+      • {location}  (table) a single `Location` or `LocationLink`
 
     Return: ~
-        (bufnr,winnr) buffer and window number of floating window or nil
+        (integer|nil) buffer id of float window
+        (integer|nil) window id of float window
```

Comment:
is there a way to express a tuple?

Reply:
this is generate from multiple return ,  tuple will like before. but it use `@retruns` I don't find `@returns` in luals annotations doc. so it might be wrong. 

```
---@return integer|nil buffer id of float window
---@return integer|nil window id of float window
```